### PR TITLE
add cloudwatch doc and link in menu to tools

### PR DESCRIPTION
**Pull request by jdmooney**

### CODE REVIEW

File: html5/_main_toc.html.slim [hunk 1]
```diff
@@ -213,6 +213,10 @@ nav.sidebar-nav
         li
           a.nav-link href="/latest/plugin_management.html"
             | Plugin Management
+        li
+          a.nav-link href="/latest/aws-tools.html"
+            | Analytic Tools
```
Comment (sbrossie) [inline html5/_main_toc.html.slim:218]: Is ` Analytic Tools` really what we want here?

Comment (jdmooney) [reply html5/_main_toc.html.slim:218]: Probably not, but I couldn't think of anything better. Do you have a suggestion?

File: userguide/tutorials/metrics-cloudwatch.adoc [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+. From the dropdown list choose the role you just created.
+
+. Click *Save*.
+
+If you have multiple instances, repeat these steps for each one. You may use the same role for all your instances.
+
+== Test the Installation
+
+To test your installation, Go to the Cloudwatch console. After a short delay, you
+should see a set of available metrics. There should be well over 1,000 in your list.
```
Comment (reshmabidikar) [inline userguide/tutorials/metrics-cloudwatch.adoc:145]: Do we need to tell users how to "**Go to the Cloudwatch console**" or is this something very obvious and we can assume users are aware of it?

File: html5/_main_toc.html.slim (hunk 1 above)
Comment (reshmabidikar) [reply html5/_main_toc.html.slim:218]: How does `AWS Analytics Tools` sound?

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've added a few minor comments.

File: userguide/tutorials/metrics-cloudwatch.adoc (hunk 2 above)
Comment (sbrossie) [reply userguide/tutorials/metrics-cloudwatch.adoc:145]: I would think they can google it up if this is not obvious.

**Comment by sbrossie** [comment]
> How does AWS Analytics Tools sound?

I am gonna merge as-is so we can see how it looks with respect to other headers, and then we can revisit.

**Comment by sbrossie** [comment]
@reshmabidikar Ok, so I see the new section:
Screen Shot 2022-03-28 at 4 00 24 PM

@jdmooney How much of this doc is specific to AWS? I think if this is just generic for KB to integrate with analytics tools, we should try to keep it as generic as possible (the doc content, and the title). However, if this tightly coupled to AWS, then we should indeed have the `AWS Analytics Tools` suggested by Reshma. I am wondering if at some point we should not just have a top level tab with all our AWS doc at some point
;

**Comment by jdmooney** [comment]
@stephane my understanding is that these documents are based on the fact 
that the AWS AMI has been preconfigured with the agent software needed 
to forward metrics and logs to these tools. They may be usable by other 
KB installations but the process would be entirely different.

**Comment by sbrossie** [comment]
@jdmooney Ok, in that case I suggest we follow @reshmabidikar 's suggestion to call it `WS Analytics Tools` - and at some point we should regroup all these AWS doc together - if this is something you are interested in looking at, you could make a proposal on how this would work.

**Comment by jdmooney** [comment]
I agree and I will use this title going forward.

File: html5/_main_toc.html.slim (hunk 1 above)
Comment (jdmooney) [reply html5/_main_toc.html.slim:218]: OK. Changed.

Comment (jdmooney) [reply html5/_main_toc.html.slim:218]: OK -- changed.

**Pull request merged**